openwisp_notifications/templates/configurables/message.md

```diff
@@ -0,0 +1,5 @@
+{% block head %} {{ level }} : {{ verb }} {% endblock head %}
```

Comment:
I'd change the path of this file to:

`templates/notifications/default_message.md` or `templates/openwisp_notifications/default_message.md`.

`configurables` is unusual.